variants/CATENA_461x/variant.h

```diff
@@ -152,14 +146,24 @@ enum {
 //
 // SERIAL_PORT_HARDWARE_OPEN  Hardware serial ports which are open for use.  Their RX & TX
 //                            pins are NOT connected to anything by default.
-#ifdef USBCON
+#if defined (USBCON) && defined (USB_TWO_HWSERIAL)
```

Comment:
```suggestion
#if defined(USBCON) && defined(USB_TWO_HWSERIAL)
```
We don't uses spaces here in MCCI code.